src/components/Pages/workshopOverview/WorkshopOverview.stories.js

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable max-len */
 import React from 'react';
-import Comments from './Comments';
+import FeedbackSection from './FeedbackSection';
 
 export default {
-	component: Comments,
-	title: 'Comments',
+	component: FeedbackSection,
+	title: 'FeedbackSection',
 };
 
-// Have an issue rendering comments because the component relies on state
+// Have an issue rendering Feedback because the component relies on state
 // We could create a new component for the comment and pass in object as prop (then we can mock the object)
```

Comment:
You can pass in props and to mock data. Check out WorkshopHeader.stories.js

Reply:
Nice I think I might have fixed this!